src/pages/workspace/AccessOrNotFoundWrapper.tsx

```diff
@@ -103,12 +103,15 @@ function PageNotFoundFallback({policyID, shouldShowFullScreenFallback, fullPageN
 }
 
 function AccessOrNotFoundWrapper({accessVariants = [], fullPageNotFoundViewProps, shouldBeBlocked, ...props}: AccessOrNotFoundWrapperProps) {
-    const {policy, policyID, report, iouType, allPolicies, featureName, isLoadingReportData} = props;
+    const {policyID, reportID, iouType, allPolicies, featureName} = props;
```

Comment:
In the current state we can move this spread to component argument (so that `...props` isn't needed anymore)